src/blueprints/tv2/value-objects/tv2-layers.ts

```diff
@@ -25,7 +28,12 @@ export enum Tv2AtemLayer {
   LOOKAHEAD = 'atem_aux_lookahead',
   DOWNSTREAM_KEYER = 'atem_dsk',
   DVE = 'atem_dve',
-  DVE_BOXES = 'atem_dve_boxes'
+  DVE_BOXES = 'atem_dve_boxes',
+}
+
+export enum Tv2VideoMixerLayer {
+  AR = 'aux_ar',
+  VIZ_OVL_AUX = 'aux_viz_ovl_in_1'
```

Comment:
OVL and AUX could be fully written out such that this becomes `VIZ_OVERLAY_AUXILIARY`. Same goes for other enum keys.

Reply:
Updated the Viz OVL entry.